geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -61,6 +61,7 @@ case class ScanPlan(filter: HBaseFilterStrategyType,
                     remoteFilters: Seq[HBaseFilter] = Nil,
                     resultsToFeatures: Iterator[Result] => Iterator[SimpleFeature]) extends HBaseQueryPlan {
   override def scan(ds: HBaseDataStore): CloseableIterator[SimpleFeature] = {
+    ranges.foreach { range => range.setAuthorizations(ds.auths) }
```

Comment:
this might not matter, but i wonder if there's some way to not have to iterate over the ranges again here